caffe2/CMakeLists.txt

```diff
@@ -503,6 +503,7 @@ endif()
 list(APPEND TORCH_SRCS
   ${TORCH_SRC_DIR}/csrc/autograd/FunctionsManual.cpp
   ${TORCH_SRC_DIR}/csrc/utils/out_types.cpp
+  ${TORCH_SRC_DIR}/library_stable_impl.cpp
```

Comment:
> Should I add a stable def and fragment when those can be done in python?

C++-only use cases need a def in C++. idk how common it is to have a def in C++ nowadays. If anyone is still using TorchScript, then they would want a def in C++.

Reply:
thanks--i've added both